src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,71 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String COMMA_SPLITTER = ",";
+    private static final String SPACE = " ";
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private final int[] types = new int[2];
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String table = readFile(fromFileName);
+        String text = writeToString(processData(table));
+        writeToFile(text, toFileName);
+    }
+
+    public String readFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                stringBuilder.append(line).append(SPACE);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read this file", e);
```

Comment:
To make this message more informative it will be better to add some details, file name for example